lib/tasks/upload_source_strings_for_translation.rake

```diff
@@ -23,29 +23,33 @@ namespace :i18n_sonder do
 
       klass.in_batches.each do |batch|
         batch.each do |obj|
-          # Check if model has the method defined and if it evaluates to true
-          model_allowed_for_translation = !klass.method_defined?(:allowed_for_translation?) ||
-              obj.allowed_for_translation?
-
-          if model_allowed_for_translation
-            # Get translated attributes, and each attribute's params for uploading translations
-            translated_attribute_names = klass.translated_attribute_names
-            translated_attribute_params = {}
-            translated_attribute_names.each do |attribute_name|
-              upload_options = obj.public_send("#{attribute_name}_backend").options[:upload_for_translation]
-              translated_attribute_params[attribute_name] = upload_options.is_a?(Hash) ? upload_options : {}
+          begin
+            # Check if model has the method defined and if it evaluates to true
+            model_allowed_for_translation = !klass.method_defined?(:allowed_for_translation?) ||
```

Comment:
@apfong once you are done with https://github.com/Flatbook/i18n/pull/39, I would like to refactor this to use the new helper module/class. That will clean this up a bit, all it needs is a switch between synchronously/async'ly calling the `perform` function of the upload worker.